logger/logging_config.py

```diff
@@ -0,0 +1,18 @@
+import json
+from logging.config import dictConfig  # type: ignore
+
+done_setup = False
+
+
+def setup_logging():
+    global done_setup
+
+    if not done_setup:
+        try:
+            logging_config_file_path = 'logger/logging_config.json'
```

Comment:
Yes, completely right, I was going to set it up with a custom config file but wasn't required for the scope of the chore and time was an issue so I just removed it. You're totally right, will remove. Cheers